### DISCUSSION TITLE
mqtt - iobroker

### DISCUSSION THREAD

**Discussion by ghoethe** [jomjol/AI-on-the-edge-device#707] [Q&A] [answered]
Ich habe die mqtt-Funktion aktiviert und im ioBroker den mqtt Adapter laufen lassen wollen, doch dieser kommt sich mit dem Sonoff-(mqtt)-Adapter in die Quere.

Also versucht über den Sonoff-Adapter den Wasserzähler einzubinden. Der Sonor Adapter erkennt den Wasserzähler, ich bekomme da aber nur einen "alive" Status übertragen, kein value etc. 

Bedarf es da eine Weile, bis genug Änderungen im Value vorhanden sind, damit dieser Wert per mqtt übertragen wird?

**Comment by jomjol** [comment, marked as answer]
Nein - entweder ein Value oder ein Fehlerwert sollte nach wenigen Minuten (nach dem ersten kompletten Durchlauf) übertragen werden.

**Reply by ghoethe** [reply]
ok, hab gesehen das der mqtt und der sonoff adapter den gleichen Port wollten, hab das umgestellt und nun werden die Daten korrekt in den matt-adapter übermittelt

** Nochmal vielen Dank für dieses Tolle Projekt! **
